src/legacy/core_plugins/table_vis/public/shim/table_vis_legacy_module.ts

```diff
@@ -0,0 +1,55 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import { once } from 'lodash';
+
+// @ts-ignore
+import { uiModules } from 'ui/modules';
+
+import 'angular-recursion';
+import 'ui/directives/paginate';
+
+// @ts-ignore
+import { TableVisController } from '../table_vis_controller.js';
+// @ts-ignore
+import { TableVisParams } from '../table_vis_params';
+// @ts-ignore
+import { KbnAggTable } from '../agg_table/agg_table';
+// @ts-ignore
+import { KbnAggTableGroup } from '../agg_table/agg_table_group';
+// @ts-ignore
+import { KbnRows } from '../paginated_table/rows';
+// @ts-ignore
+import { PaginatedTable } from '../paginated_table/paginated_table';
+
+/** @internal */
+export const initTableVisLegacyModule = once((): void => {
+  uiModules
+    .get('kibana/table_vis', ['kibana'])
+    .controller('KbnTableVisController', TableVisController)
+    .directive('tableVisParams', TableVisParams);
+
+  // todo: not sure that "kibana" is a right module for that directives
```

Comment:
Not sure if this was meant to be a todo before checkin or not but seems like something to resolve before merging

Reply:
You might be able to get away with using `kibana/table_vis` for everything since all of these directives are specific to the table vis at this point i think.

@lizozom do you have any insight on this?

Reply:
@lukeelmers  @lizozom  @nreese  checked it again and looks like we can move all into one module without any risk. PR was updated